<a href="https://colab.research.google.com/github/ragedharb16/sales-predictions/blob/main/project_2_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Raged abou 

03/10/2022

Option A

In [2]:
import pandas as pd 
import numpy as np

In [5]:
df= pd.read_csv('/content/outwithoutindex.csv')
df.shape

(6825, 16)

1) Source of data

https://www.kaggle.com/jonhcarvalho/videogames

2) Brief description of data

the data is about video games is divided in genres, plataform, critics, ratings, developers among others , is one of the biggest industry in the world 

3) What is the target? 

the target will be global sales

4) Is this a classification or regression problem?

is a regression problem since we want to predict numbers 

5) How many features?


it has 15 features and one target

6) How many rows of data.

6825 rows

7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

well acoording to cleaning is a big data we might have some typen problems or duplicated we going to look into that, there is visualitation challenges, and of course with machine learning that we going to aproach using diferents methods